DQM/DTMonitorModule/python/dtTriggerSynchTask_cfi.py

```diff
@@ -27,4 +28,8 @@
     tTrigMode = cms.string('DTTTrigSyncFromDB')
 )
 
+from Configuration.Eras.Modifier_run2_DT_2018_cff import run2_DT_2018
+run2_DT_2018.toModify(dtTriggerSynchMonitor,processDDU = cms.bool(False))
```

Comment:
this has to be simply ```processDDU = False```
type specs are not needed
https://hypernews.cern.ch/HyperNews/CMS/get/tier0-Ops/1875/1/1.html